custom_components/midea_ac_lan/__init__.py

```diff
@@ -258,3 +259,66 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     # Forward the unloading of an entry to platforms
     await hass.config_entries.async_unload_platforms(config_entry, ALL_PLATFORM)
     return True
+
+
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug(
+        "Migrating configuration from version %s.%s",
+        config_entry.version,
+        config_entry.minor_version,
+    )
+
+    # 1.0 -> 1.1:  convert device identifiers from int to str
+    if config_entry.version == 1 and config_entry.minor_version == 0:
```

Comment:
where is this version coming from? I don't see it defined in your config flow class

Reply:
Agreed, default `version` and `minor_version` are both `1`, not `0`,
and the minor version was introduced in 2024.1
This is a breaking change, so l recommend upgrading `version`.

Reply:
I think we can completely ignore minor_version if we want keep compatibility with old HA.